src/media.rs

```diff
@@ -90,4 +114,12 @@ mod tests {
       r"unsupported file extension `\.foo`, supported extensions: apng .*"
     );
   }
+
+  #[test]
+  fn check_mp4_codec() {
+    assert!(
+      Media::check_mp4_codec(Path::new("bitcoin-pup.mp4")).is_ok(),
```

Comment:
We could use a negative test too, to check that an MP4 with an unsupported video codec fails the check.

Reply:
I'll look into it.